psrecord/main.py

```diff
@@ -209,6 +209,9 @@ def monitor(pid, logfile=None, plot=None, duration=None, interval=None,
 
     if plot:
 
+        # Use non-interactive backend, to enable operation on headless machines.
+        import matplotlib as mpl
+        mpl.use('Agg')
```

Comment:
Rather than changing it globally, just in case anyone uses psrecord from within a Python/IPython session, could you instead use a context manager, i.e.

```
with plt.rc_context({'backend': 'Agg'}):
    import matplotlib.pyplot as plt
    # plotting code
```

Thanks!